pithos/pithos.py

```diff
@@ -1128,6 +1133,10 @@ def on_configure_event(self, widget, event, data=None):
         self.settings.set_value('win-pos', GLib.Variant('(ii)', (event.x, event.y)))
         return False
 
```

Comment:
And connect the signal handler in https://github.com/pithos/pithos/blob/master/data/ui/PithosWindow.ui